api/server/v1/tx.go

```diff
@@ -51,10 +51,11 @@ const (
 	DescribeDatabaseMethodName   = apiMethodPrefix + "DescribeDatabase"
 	DescribeCollectionMethodName = apiMethodPrefix + "DescribeCollection"
 
-	ObservabilityMethodPrefix = "/tigrisdata.observability.v1.Observability/"
-	ManagementMethodPrefix    = "/tigrisdata.management.v1.Management/"
-	CreateNamespaceMethodName = ManagementMethodPrefix + "CreateNamespace"
-	ListNamespaceMethodName   = ManagementMethodPrefix + "ListNamespaces"
+	ObservabilityMethodPrefix    = "/tigrisdata.observability.v1.Observability/"
+	ManagementMethodPrefix       = "/tigrisdata.management.v1.Management/"
+	CreateNamespaceMethodName    = ManagementMethodPrefix + "CreateNamespace"
+	ListNamespaceMethodName      = ManagementMethodPrefix + "ListNamespaces"
+	DescribeNamespacesMethodName = ManagementMethodPrefix + "DescribeNamespaces"
```

Comment:
Good point on making it consistent. Should the plural be ListSomething and the singular be DescribeSomething? That would be consistent with other databases.

Reply:
But Database and Collection are specific to a single database and collection. Here we are saying that it will dump information for all the namespaces or a single one.